src/index.html

```diff
@@ -146,6 +146,262 @@
          document.getElementById("main").style.marginLeft= "0";
      }
     </script>
+
+    <svg>
+        <defs>
+            <!-- arrowhead marker definition -->
+            <marker id="arrow" viewBox="0 0 10 10" refX="5" refY="5"
+                markerWidth="4" markerHeight="4"
+                orient="auto-start-reverse">
+            <path d="M 0 0 L 10 5 L 0 10 z" />
+            </marker>
+
+            <!-- simple dot marker definition -->
+            <marker id="dot" viewBox="0 0 10 10" refX="5" refY="5"
+                markerWidth="5" markerHeight="5">
+            <circle cx="5" cy="5" r="5" fill="red" />
+            </marker>
+        </defs> 
+        <defs id="symbols">
```

Comment:
maybe add a comment here to say that all these symbols are required/used in the reviewGraph visualisation. Similarly, in the reviewGraph, where these are called/referred to, add a comment saying that these symbols are defined in the `src/index.html`, which should help future maintainers to find their way around the code.

Reply:
Added a comment in src/index.html; there was already a [comment in reviewGraph.js.](https://github.com/flaviomerenda/dashboard-fe/blob/review-graph-dev/src/app/controllers/reviewGraph.js#L554)